### PR TITLE
feat: add playwright exmaple

### PR DESCRIPTION
**Pull request by irrationnelle**
Playwright is recently considered interesting automate tool for browser like selenium, puppeteer.
But there is no example for Playwright yet, so I suggest this PR.
IMHO, this example is very simple and quintessential, but any feedback could improve also this PR.
Thank you!

### CODE REVIEW

File: packages/playwright/.mocharc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (craigtaub) [inline packages/playwright/.mocharc.json:1]: You can probably remove the values which are the default, i.e. everything except `timeout`.

**Review by craigtaub — COMMENTED** [review]
Thanks, nice example 👍🏻

**Comment by irrationnelle** [comment]
Thank you for feedback!

`.mocharc.json` is removed and timeout option is moved to package.json.

and to pass Travis CI, I modified below:

- add timeout option with `10000` value for `selenium` and `vue-puppeteer`

- add `Promise` to async operation of `node-sqlite3`, because sometimes test is failed against non-existent value. Check [here](https://travis-ci.org/github/mochajs/mocha-examples/builds/721743121). IMHO, this would be an asynchronous problem, for example 1) before insert value, request value from table 2) first create table and then drop table. I would like to ask @MoonSupport to confirm my idea 🙏.

**Comment by irrationnelle** [comment]
I missed something out.  

`chromedriver` is updated with `^85.0.0` to pass Travis CI that installed chrome v85 as stable version for `selenium` example.

**Pull request merged**